srcpkgs/textadept/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'textadept'
+pkgname=textadept
+version=10.6
+revision=1
+archs="i686 x86_64"
+wrksrc=textadept-textadept_${version}
+hostmakedepends="autoconf automake gettext libtool clang wget unzip darcs pkg-config glib-devel ncurses"
+makedepends="gtk+-devel ncurses-devel"
+depends="gtk+ lua52"
+short_desc="Fast, minimalist, and extensible text editor for programmers"
+maintainer="reback00 <reback00@protonmail.com>"
+license="MIT"
+homepage="https://foicica.com/textadept/"
+distfiles="https://github.com/rgieseke/textadept/archive/textadept_${version}.tar.gz"
+checksum=2cb2aa8266c166ed4a3f534843aa75cec0dc1ced09c007aaef540e3a7999870b
+
+pre_configure() {
+	cd src
```

Comment:
This is better to include in wrksrc imho, even if xlint will complain.

@leahneukirchen, do you remember reason for discouraging deep wrksrc?

Reply:
I have changed wrksrc to `.../src` and travis is complaining:

```
srcpkgs/textadept/template:5: wrksrc should be a top-level directory
```

Will it be a problem?

Reply:
Well, John explained me it's wrong, leave wrksrc as is and set build_wrksrc=src.

Reply:
I added it and I got all [Green checks](https://travis-ci.org/void-linux/void-packages/builds/599279593). But then I saw this in Manual:
> `create_wrksrc` ... **Required** if a package contains multiple distfiles.

I added `create_wrksrc=yes` and it fails in some [builds](https://travis-ci.org/void-linux/void-packages/builds/599285759) with:

> => textadept-10.6_1: extracting distfile(s), please wait...
> /void-packages/common/xbps-src/shutils/common.sh: line 33: cd: src: No such file or directory
> => ERROR: textadept-10.6_1: cannot access build_wrksrc directory [src]
> The command "if [ -z "$XLINT" ]; then docker exec -t void hostrepo/common/travis/build.sh $BOOTSTRAP $ARCH; fi" exited with 1.

Is `create_wrksrc` absolutely necessary? or is there a fix I can apply?

Reply:
Not required in this case + https://github.com/void-linux/void-packages/pull/15278#discussion_r335131904